http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/AppMonitorConfig.scala

```diff
@@ -13,5 +13,6 @@ case class AppMonitorConfig(nodepoolCreate: PollMonitorConfig,
                             scalingUpNodepool: PollMonitorConfig,
                             scalingDownNodepool: PollMonitorConfig,
                             startApp: InterruptablePollMonitorConfig,
-                            updateApp: InterruptablePollMonitorConfig
+                            updateApp: InterruptablePollMonitorConfig,
+                            appLivelinessCheck: PollMonitorConfig
```

Comment:
Hmmm.. this isn't a verb though its a noun, the `appLivelinessCheck[Config]`

Reply:
Why not a verb? all the rest of these are verbs

Reply:
well, except for the `scalingDown|Up` which for some reason are gerunds

Reply:
They aren't though, a verb implies it is a method that performs an action. `updateApp` could be interpreted in this ambiguous form as an action descriptor, but it really is the `updateAppConfig`, with the `config` part being omitted due to redundancy/variable name length.

Reply:
isn't "update app config" a verb with the object of the verb being the "app config"?

Nevertheless I think what you're saying is that this is a GET endpoint, not a mutative action, so it's not doing anything but retrieving a status. These usually get named things like `/status` or `/details` etc, and calling them `getDetails` is redundant. In that case I'd advocate for `appLiveliness` since the verb isn't important.

Reply:
I'm fine with `appLiveness` here. Though for the sake of curiosity, I do believe `updateApp` is a compound adjective describing `config` in this case.

Reply:
oh, I see what you meant :) I was misreading when you talked about config here. Yeah, i get what you're saying, all these are the names of different config blocks tied to the various actions: updateApp, startApp etc.